src/workers.ts

```diff
@@ -50,8 +50,10 @@ if (SHOULD_LOG_EVENTS) {
   console.info('Loading penumbra script element...');
 }
 let scriptElement: HTMLScriptElement = (document.currentScript ||
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
so rather than eslint disabling everywhere, i normally just leave these as eslint warns since it normally means you are trying to get around the type system incorrectly. 

i leave it as the warns so it acts like a TODO to fix the type